project-base/storefront/components/Forms/Radiobutton/Radiobutton.tsx

```diff
@@ -12,25 +12,22 @@ export type RadiobuttonProps = NativeProps & {
     value: any;
     checked: InputHTMLAttributes<HTMLInputElement>['checked'];
     label: ReactNode;
-    onChangeCallback?: (newValue: string | null) => void;
+    onClick?: (newValue: string | null) => void;
```

Comment:
In commit message I see 

_"radiobutton component now does not accept onClick prop
anymore, so you should provide your actions and callbacks
via onChange"_

But here I see that it still accept this prop...?

Reply:
Oh sheesh, you are right. This was done in a fixup and did not change the message. Superb work! Thanks for taking such a detailed look 💙 I rewrote the message so it actually matches what was done.